### ISSUE TITLE
Cannot jump back after ":Semshi goto <anything>"

### ISSUE THREAD

**Issue by akrejczinger** [numirias/semshi#62]
Vim has a jump stack which can be used to jump to the previous location, as shown here:
https://vim.fandom.com/wiki/Jumping_to_previously_visited_locations
This works by default when using tag files and jumping between tags, and also with some Vim movements (like going to the end of file).
When using `:Semshi goto name` or the other goto functions, the jump stack is not updated, so I cannot jump back to the original location with the default Ctrl+O mapping.

**Comment by numirias** [comment]
Yep, these jumps should totally go into the jump list. Will look into it.
